sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GatewayAddressCache.java

```diff
@@ -99,6 +100,7 @@ public class GatewayAddressCache implements IAddressCache {
     private IOpenConnectionsHandler openConnectionsHandler;
     private final ConnectionPolicy connectionPolicy;
     private final boolean replicaAddressValidationEnabled;
+    private final List<Uri.HealthStatus> replicaValidationScopes;
```

Comment:
This could be a set, since it only takes enums.

Reply:
updated